client/me/help/help-contact-form/index.jsx

```diff
@@ -109,6 +112,17 @@ export class HelpContactForm extends React.PureComponent {
 		qanda: [],
 	};
 
+	componentWillMount() {
+		const { npsSurveyFeedback, translate } = this.props;
+
+		if ( npsSurveyFeedback ) {
+			this.state.message =
+				'\n' +
```

Comment:
Why do we need this newline at the beginning? In my case, I couldn't see what I had entered because of it:
![image](https://user-images.githubusercontent.com/1842898/61109291-221ff180-a4b7-11e9-9d74-d693e5889743.png)

Reply:
The blank line on the top is padding, so users can write their comment without typing enter. It's like the one we have on response emails.

Reply:
Cool. Thanks for the clarification.